torch/_inductor/simple_fsdp.py

```diff
@@ -1,136 +0,0 @@
-# mypy: allow-untyped-defs
```

Comment:
this file should be delete/moved in the next PR on reordering, so that differences between old work and new work can be compared and visualized.